tests/cpp/test_utils.cpp

```diff
@@ -2072,4 +2074,84 @@ TEST_F(TestCpp23BackPort, Enumerate) {
   // bidirectional
 }
 
+namespace {
+
+// Generator that yields integers from 0 to n-1
+Generator<int> count(int n) {
+  for (int i = 0; i < n; ++i) {
+    co_yield i;
+  }
+}
+
+// Generator that yields integers from n to 2*n - 1
+Generator<int> count2(int n) {
+  for (int i = n; i < 2 * n; ++i) {
+    co_yield i;
+  }
+}
+
+// Generator that yields integers from 0 to 2*n - 1
```

Comment:
I believe this is a copy-and-paste error. The comment doesn't mention `m` at all.

Reply:
It is! fixed